pom.xml

```diff
@@ -114,23 +114,22 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>8</maven.compiler.source>
+    <maven.compiler.target>8</maven.compiler.target>
+    <maven.compiler.release>8</maven.compiler.release>
     <additionalparam>-Xdoclint:none</additionalparam>
     <!-- Dependency versions -->
     <junit.version>4.13.2</junit.version>
     <junit-jupiter.version>5.8.2</junit-jupiter.version>
     <mockito.version>4.6.1</mockito.version>
     <!-- Plugin versions -->
-    <findbugs-maven-plugin.version>3.0.5</findbugs-maven-plugin.version>
+    <spotbugs-maven-plugin.version>4.7.1.0</spotbugs-maven-plugin.version>
     <jacoco-maven-plugin.version>0.8.8</jacoco-maven-plugin.version>
-    <javancss-maven-plugin.version>2.1</javancss-maven-plugin.version>
-    <jdepend-maven-plugin.version>2.0</jdepend-maven-plugin.version>
```

Comment:
@joel-costigliola @fbiville what's your view about the removal of these plugins? This will impact other AssertJ projects which may still be using them, such as AssertJ Neo4J.

Reply:
I don't actively use these reports so for me, they can be removed

Reply:
same here, we can remove their usage from the Neo4j module